cmd/manager/main.go

```diff
@@ -29,13 +29,16 @@ import (
 
 // Change below variables to serve metrics on different host or port.
 var (
-	metricsPort                   = "8080"
-	metricsPath                   = "/metrics"
-	secretWatcherScanInterval     = time.Duration(10) * time.Minute
-	hours                     int = 1
-	totalWatcherInterval          = time.Duration(15) * time.Minute
+	metricsPort               = "8080"
+	metricsPath               = "/metrics"
+	secretWatcherScanInterval = time.Duration(10) * time.Minute
+	hours                     = 1
+	totalWatcherInterval      = time.Duration(15) * time.Minute
 )
 
+// Set Local evn "FORCE_DEV_MODE = local" in order to run locally
+const envDevMode = "FORCE_DEV_MODE"
```

Comment:
Maybe I am missing something but couldn't we inject this value into another package? that way we don't need to retrieve it through `os` i.e. `utils.envDevMode="FORCE_DEV_MODE"`? I would even consider using a boolean if testing is the only "DevMode" we use. `utils.testMode`?

Reply:
I'm not completely sure what you're suggesting when you say 'injecting into another package'.

Do you mean something like editing `utils.go` and adding a line like `testMode := false` at the top of the file and changing that line when you're trying to run it locally?

Reply:
@iamkirkbater Yes

Reply:
I am way more stoked than I should be for this

Reply:
The only problem I have with that is then you start modifying the code again to change the flag.  Part of the idea here is that we don't want to physically modify the code before committing or after running our tests. 

---

With that said, though, I don't think it would be a _bad_ idea to pull this functionality out to somewhere like `utils.go` but still utilize something like an environment variable or command option there, and then the rest of the codebase will just do something like `if ! utils.TestMode`.

So in that case in `utils.go` we would have `TestMode := os.getEnv('FORCE_DEV_MODE')` instead of setting the same variable and such throughout the codebase.

Reply:
I see, then I agree with you. Move it to utils but keep it the same